Dropdown/Dropdown.module.less

```diff
@@ -143,35 +125,122 @@
 							border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
 							border-top-color: @agate-dropdown-button-open-border-color;
 						}
+
+						.focus({
+							.bg {
+								border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
+							}
+						})
 					}
 				}
+			}
+		}
 
-				.dropdownList {
-					border-radius: @agate-dropdown-up-button-list-border-radius;
-				}
+	});
+}
+
+.dropdownList {
+	.applySkins({
+		border-bottom: @agate-dropdown-border-width @agate-dropdown-border-style @agate-dropdown-border-color;
+		border-radius: @agate-dropdown-list-border-radius;
+		background-color: @agate-dropdown-list-bg-color;
+		overflow: hidden;
+
+		.item {
+			border-color: @agate-item-border-color;
+			border-width: @agate-item-border-width;
+
+			.focus({
+				border-color: @agate-item-focus-border-color;
+			});
+
+			&.selected {
+				color: @agate-item-color;
 			}
+		}
 
-			.dropdownListWithScroller {
-				.scroller {
-					// FIXME:
-					[role="button"] {
-						.focus({
-							& > div {
-								background-image: none;
-							}
-						});
-					}
+		.dropDownListItem {
+			font-size: @agate-dropdown-item-font-size;
+			font-weight: @agate-dropdown-item-font-weight;
+			line-height: @agate-dropdown-item-line-height;
 
-					[role="button"] > div {
-						border-color: transparent;
-						color: black;
-					}
+			&.radioItem {
+				border-color: @agate-dropdown-item-border-color;
+				border-width: @agate-dropdown-item-border-width;
+				padding-left: @agate-dropdown-item-padding;
+				padding-right: @agate-dropdown-item-padding;
 
-					[role="button"] + div::before {
-						background-color: black;
-					}
+				.focus({
+					background-color: @agate-dropdown-item-focus-bg-color;
+				});
+			}
+		}
+
+		&.smallest {
+			width: @agate-dropdown-list-smallest-width;
+		}
+
+		&.small {
+			width: @agate-dropdown-list-small-width;
+		}
+
+		&.medium {
+			width: @agate-dropdown-list-medium-width;
+		}
+
+		&.large {
+			width: @agate-dropdown-list-large-width;
+		}
+
+		&.x-large {
+			width: @agate-dropdown-list-x-large-width
+		}
+
+		&.huge {
+			width: @agate-dropdown-list-huge-width;
+		}
+
+		&.above {
+			border-radius: @agate-dropdown-up-button-list-border-radius;
+		}
+
+		&.dropdownListWithScroller {
+			height: @agate-dropdown-list-height;
+
+			.scroller {
+				height: 100%;
+				// FIXME:
+				[role="button"] {
+					.focus({
+						& > div {
+							background-image: none;
+						}
+					});
+				}
+
+				[role="button"] > div {
+					border-color: transparent;
+					color: black;
+				}
+
+				[role="button"] + div::before {
+					background-color: black;
```

Comment:
What are these code for?

Reply:
This was the only solution that could be found a while ago to customize the Scroller inside DropdownList to match the UX design for Silicon

Reply:
Does the current style of scrollbar button of Scroller meet the GUI requirement?
![image](https://user-images.githubusercontent.com/5037429/108472914-b0ce3480-72d0-11eb-9f11-27b01d117bd6.png)
If not, I guess we need to fix it by Scrollbutton side.

Reply:
AVN 1.5 GUI requirement did not have any focus styling for Scroller buttons, only for a Scroller inside a Dropdown. That;s why we eneded up with the styling above.
But we are looking forward tothe AVN 2.5 GUI requirements for Scroller. The ones for Dropdown are released and are different than these ones above. We will change them 100% in the future user story for Dropdown styling updates. Until then, let's keep these ones :)